ide/coqide/FAQ.md

```diff
@@ -0,0 +1,81 @@
+# CoqIDE FAQ #
+
+### What is CoqIDE?
+
+A powerful GTK3-based graphical interface for Coq.
+See https://coq.inria.fr for more information.
+
+### How to enable Emacs keybindings?
+
+Insert the following line in your `gtkrc` file:
+```
+gtk-key-theme-name = "Emacs"
+```
+The location of this file is system-dependent. If you're running
+GNOME, you may use the graphical configuration tools.
+
+See also the µPG mode in CoqIDE preferences and help.
+
+### How to enable antialiased fonts?
+
+Set the `GDK_USE_XFT` variable to 1. This is the default with Gtk >= 2.2.
+If some of your fonts are not available, set `GDK_USE_XFT` to 0.
```

Comment:
The comment about GTK 2.2 is from 2003, and as it says, it is the default, so not sure this is worth mentioning.

The antialiasing in GTK is not very good.  Here is some regular and enlarged CoqIDE text compared to Word.  The CoqIDE text looks bad and changing the setting to 0 doesn't change the appearance.  (I wonder whether this switch still exists in the GTK source.)

![image](https://github.com/coq/coq/assets/1253341/e3bff22b-5f73-4a1a-8475-12e8d5229e22)

![image](https://github.com/coq/coq/assets/1253341/24dbbaf7-2a3c-4860-8709-dfc770af8b8e)

![image](https://github.com/coq/coq/assets/1253341/8d86b870-786c-4766-a960-cbf7390f6eb5)

![image](https://github.com/coq/coq/assets/1253341/8dca1fd1-9d3d-4ae6-bce0-d2fa69e5e34c)

Reply:
Thank you for your analysis. I’ve removed the corresponding section.